lib/ingreedy/ingreedy_parser.rb

```diff
@@ -112,13 +120,22 @@ def parse
       parslet = super(original_query)
 
       result[:amount] = rationalize parslet[:amount]
-      result[:amount] = [result[:amount], rationalize(parslet[:amount_end])] if parslet[:amount_end]
+      result[:amount] = [
+        result[:amount],
+        rationalize(parslet[:amount_end]),
+      ] if parslet[:amount_end]
+
       result[:container_amount] = rationalize(parslet[:container_amount])
 
-      result[:unit] = convert_unit_variation_to_canonical(parslet[:unit].to_s) if parslet[:unit]
-      result[:container_unit] = convert_unit_variation_to_canonical(parslet[:container_unit].to_s) if parslet[:container_unit]
+      result[:unit] = convert_unit_variation_to_canonical(
+        parslet[:unit].to_s,
+      ) if parslet[:unit]
+
+      result[:container_unit] = convert_unit_variation_to_canonical(
+        parslet[:container_unit].to_s,
+      ) if parslet[:container_unit]
 
-      result[:ingredient] = parslet[:ingredient].to_s.lstrip.rstrip #TODO cheating
+      result[:ingredient] = parslet[:ingredient].to_s.lstrip.rstrip # TODO hack
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.